portfolio/src/main/java/com/google/sps/servlets/FactsServlet.java

```diff
@@ -8,11 +8,11 @@
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/facts")
-public class HelloWorldServlet extends HttpServlet {
+public class FactsServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("I am an identical twin!");
```

Comment:
I think this is supposed to return an `ArrayList` of facts, using GSON to convert to JSON.
Then in your `showFacts` function, you can return a random fact instead of just one fact.

Let me know if you have any questions about that!

Reply:
This commit is for the fetch tutorial! I'm currently working on the JSON tutorial.

Reply:
Sorry! I got confused

Reply:
Spoiler alert!!